### ISSUE TITLE
Sign up/up feedback

### ISSUE THREAD

**Issue by AlesteCapital** [Polkadex-Substrate/Polkadex-Open-Beta#68]
Signing In
1. Signing in should be as simple as: selecting account in Polkadot.js & Allowing the application in the Polkadot.js wallet pop out screen. 
2. The proxy account should be set up automatically and use the main account as the access key. - This way there is no need for the user to store additional mnemonic phrases or exposing them on the browser. 
3. If you want more proxy accounts later, then there should be a simple + button that’s adds it.  (Again the main account should be used as the access key). 

The sign in process should be simple & fast without the need to right down additional phrases. (Similar to how Ledger wallets work - one master for all)

Like with a CEX, the account signed in with should be labelled as main account, & the first proxy account should be labelled as trade account.  Any additional proxy accounts should be known as sub accounts & the user can choose the name of them.  

Ditch the label ‘proxy account’ - it may confuse people. 

First Attempt Logging in: 
Proxy account was created, but then when trying to sign in we saw this: Error: This proxy account has not been registered yet!

Second Attempt:
We tried to create a new account to gain access, but then it said the main account has already been registered.

**Comment by Leowarrior911** [comment]
I totally agree. I have tried numerous times I gave up trying to log in. It should be as simple as a click like we connect to our Metamask.

**Comment by jaymansfield** [comment]
+1

Definitely could be easier

**Comment by AlesteCapital** [comment]
When you say _'This is because all the trades are signed by the proxy accounts keyring.'_ - Am I right in thinking that if you have this proxy account, you wont need to manually sign every action when using the platform?

If that is the case then I understand the reasoning behind the proxy account; however this still needs to be implemented in a way that doesn't require the user doesn't need to take note of any new keys, or have them exposed in your web browser initially.  

web3 access should be as simple as: connect wallet > use functionality. - The process needs to be streamlined else ease of access will be the first user deterrent. 

Remember Polkadex is supposed to be cutting edge solution.
